js/siteorigin-panels/view/builder.js

```diff
@@ -500,28 +503,18 @@ module.exports = Backbone.View.extend( {
 			defaultPosition: 'first'
 		}, options );
 
-		if ( this.$( '.so-cells .cell' ).length === 0 ) {
-
+		if( ! this.model.get('rows').length ) {
+			// There aren't any rows yet
 			if ( options.createCell ) {
 				// Create a row with a single cell
 				this.model.addRow( [{ weight: 1 }], { noAnimate: true } );
 			} else {
 				return null;
 			}
-
-		}
-
-		var activeCell = this.$( '.so-cells .cell.cell-selected' );
-
-		if ( activeCell.length === 0 ) {
-			if ( options.defaultPosition === 'last' ) {
```

Comment:
If we're not taking `defaultPosition` option into account anymore then it should probably be removed from line 503 too?